src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -1197,10 +1223,31 @@ void ViewSceneDialog::keyReleaseEvent(QKeyEvent* event)
   {
   case Qt::Key_Shift:
     shiftdown_ = false;
+    updateCursor();
     break;
   }
 }
 
+void ViewSceneDialog::focusOutEvent(QFocusEvent* event)
+{
+  shiftdown_ = false;
```

Comment:
This is buggy--when I focus out, a pointer cursor changes to open hand (which it should never be outside of a VS window). Probably just do this instead:
```suggestion
  setCursor(Qt::ArrowCursor);
```

Reply:
OK for some reason I forgot the network editor also uses the hand-grab cursor for module selection. It just seemed a bit weird with the transition. Is there a way to customize the color of the hand for ViewScene?

Reply:
Focus out only happens when you click away from the window. The [cursor doc](https://doc.qt.io/qt-5/qcursor.html) says that it only supports colored cursor for X11. Mac and Windows don't use X11 for qt apps and Linux usage is going down because it is being replaced by Wayland.

Reply:
Maybe these should be part of hoverEnter/Leave events then? We’ll discuss today at meeting